aws/resource_aws_globalaccelerator_listener.go

```diff
@@ -0,0 +1,270 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/globalaccelerator"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+)
+
+func resourceAwsGlobalAcceleratorListener() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsGlobalAcceleratorListenerCreate,
+		Read:   resourceAwsGlobalAcceleratorListenerRead,
+		Update: resourceAwsGlobalAcceleratorListenerUpdate,
+		Delete: resourceAwsGlobalAcceleratorListenerDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Timeouts: &schema.ResourceTimeout{
```

Comment:
We should likely remove these customizable timeouts since:
* They do not seem to be determined by a scalable resource such as amount of data
* They are currently undocumented 😉